include/cantera/kinetics/ReactionData.h

```diff
@@ -142,37 +156,28 @@ struct FalloffData : public ArrheniusData
  * The data container `PlogData` holds precalculated data common to
  * all `PlogRate` objects.
  */
-struct PlogData
+struct PlogData final : public ReactionData
 {
-    PlogData() : temperature(1.), logT(0.), recipT(1.), pressure(NAN), logP(0.) {}
+    PlogData() : pressure(NAN), logP(0.) {}
+    using ReactionData::update;
```

Comment:
This `using` is unnecessary if `PlogData` implements all three overloads, right? Same for `ChebyshevData`.